packages/react-magma-dom/src/components/StyledAIButton/styles.ts

```diff
@@ -0,0 +1,157 @@
+import { transparentize } from 'polished';
+
+import { AIButtonVariant } from '../AIButton';
+
+const DEFAULT_STYLE_PROPS = {
+  shape: 'fill',
+  size: 'medium',
+  textTransform: 'uppercase',
+  variant: 1,
+};
```

Comment:
If we have these default style props, should we also use them in the component? It's going to be confusing if we ever have to update the defaults because they are stored in two separate places